sklego/ensemble_model.py

```diff
@@ -0,0 +1,87 @@
+from sklearn.linear_model import LogisticRegression
+from sklearn.metrics import confusion_matrix
+
+class EnsembleModel():
+    """
+    A classifier that takes the uncertainty p(class | model) of a classifier into account.
+    Based on the requirements of issue #23 of koaning/scikit-lego
```

Comment:
It might be nice to refer to the original paper here

Reply:
It's not *that* great of a paper. But sure.

Reply:
Where can I find the paper?

Reply:
https://link.springer.com/chapter/10.1007/978-3-319-00651-2_19